Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -1059,7 +1059,7 @@ def main():  # pragma: nocover
             max_incidents_to_fetch = arg_to_number(params.get("max_fetch", FETCH_LIMIT))
             fetch_threats = params.get("fetch_threats", False)
             # Get the polling lag time parameter
-            polling_lag_minutes = int(params.get('polling_lag', 5))
+            polling_lag_minutes = int(params.get('polling_lag', 2))
```

Comment:
Consider updating the [YML](https://github.com/demisto/content/blob/master/Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml#L77) file and [README](https://github.com/demisto/content/blob/master/Packs/AbnormalSecurity/Integrations/AbnormalSecurity/README.md?plain=1#L22) file as well with the default 2 minutes

Reply:
This is done @MosheEichler